docs/sop/DEPLOYING_TO_PRODUCTION.md

```diff
@@ -50,9 +50,12 @@ aws_secret_access_key=<upstream_alt_aws_secret_access_key>
 ```
 
 The values for `upstream_aws_access_key_id`, `upstream_aws_secret_access_key`, `upstream_alt_aws_access_key_id`,
-and `upstream_alt_aws_secret_access_key` are stored in Gitlab variables for the `cngb-infra` project.
+and `upstream_alt_aws_secret_access_key` are stored in Gitlab variables for the `cngb-infra` project (we use GigaDB's keys in both case)
 
->**Note**: Do not use the same keys for both sections, as if that key led to the compromise of one infrastructure, then the other one is done for too. For similar reason, do not use the keys you are using on your AWS developers deployment.
+>**Note 1**: Do not use the keys you are using on your AWS developers deployment as that would weaken security on production environment
+>**Note 1**: In a previous version of this doc, we used two different IAM users for each deployment but that's the wrong separation of concerns
+> because active and standby are two parts of the same infrastructure. Also having different users would complicate quick switchover.
+> Finally a better separation (TODO in the future) is having different IAM users for staging environment and for live environment as staging environments are more permissive by nature.
```

Comment:
Hi @rija,

Even the upstream and upstream-alt are actually sharing the GigaDB's keys, the efs in upstream and the efs in upstream-alt are still not the same, once we run the `terrafrom destroy` and `terraform apply`, the efs will still be destroyed and the action of syncing the efs is still needed. Is there way to destroy the infra, except the efs?